flink-libraries/flink-table/src/main/java/org/apache/calcite/sql/fun/SqlGroupFunction.java

```diff
@@ -50,7 +50,8 @@
  * GROUP BY HOP(rowtime, INTERVAL '1' HOUR), productId
  * </pre></blockquote>
  */
-class SqlGroupFunction extends SqlFunction {
+// FLINK QUICK FIX
+public class SqlGroupFunction extends SqlFunction {
```

Comment:
Calcite 1.13 was released and we are planning to update the dependency. 
I guess we cannot simply drop this file and use Calcite's version, if we start modifying the code. 

Do we have plans to contribute the changes back or do we want to keep this file?